Assets/Prefabs/WallPieces/Only Wall.prefab

```diff
@@ -67,6 +67,12 @@ PrefabInstance:
       propertyPath: m_Materials.Array.data[0]
       value: 
       objectReference: {fileID: 2100000, guid: 67390ba79e2d3754881ca32e107975cb, type: 2}
+    - target: {fileID: -6400281290468069383, guid: 4e56b1dee66b3e449be01c8d422e8c19,
```

Comment:
What are you adding here? A Collider?

Reply:
I didn't add anything to it. Maybe it's because I  changed the mesh around and accidentally pushed that.